### PR TITLE
Improves Issue 59: Updates Header component

### PR DESCRIPTION
**Pull request by chrispinkney**
Should be enough to fully close issue #59

Added a few things:
* Added bootstrap icons dependency
* Changed favicon and web app description
* Added logic for login and logout buttons
* Added placeholder search bar form
* Containerized the header into 3 categories
* Added cart total placeholder code + icon

### CODE REVIEW

File: src/frontend/src/components/Header.css [hunk 1]
```diff
@@ -16,3 +16,12 @@ body {
   float: left;
   padding-right: 5px;
 }
+
+.header-search-form {
```
Comment (nilan01) [inline src/frontend/src/components/Header.css:20]: I don't think this would target the class you are trying to target.

Comment (chrispinkney) [reply src/frontend/src/components/Header.css:20]: Fixed!

Comment (chrispinkney) [reply src/frontend/src/components/Header.css:20]: FTFY

**Comment by chrispinkney** [comment]
Lookin' pretty good if I do say so myself.

image

**Review by nilan01 — APPROVED** [review]

**Pull request merged**